symplyphysics/laws/thermodynamics/second_virial_coefficient_for_van_der_waals_equation_of_state.py

```diff
@@ -0,0 +1,66 @@
+from sympy import Eq
+from symplyphysics import (
+    units,
+    Quantity,
+    Symbol,
+    Function,
+    print_expression,
+    validate_input,
+    validate_output,
+    symbols,
+)
+
+# Description
+## The second virial coefficient is a coefficient appearing in the virial equation of state of
+## a gas which describes the deviation from ideal gas behaviour. This coefficient describes pair
+## interaction between molecules of the substance and can be found if the pair interaction potential
+## is known, but it can also be derived from the equation of state as a series expansion with respect
+## to inverse molar volume or equivalently to molar density.
+
+# Law: B(T) = b - a / (R * T)
+## B(T) - second virial coefficient of the virial expansion
+## a - parameter of the van der Waals equation of state representing the magnitude of intermolecular forces
+## b - parameter of the van der Waals equation of state representing the effective molecular size
+## R - molar gas constant
+## T - absolute temperature
+
+second_virial_coefficient = Function("second_virial_coefficient", units.volume / units.amount_of_substance)
```

Comment:
I think we've agreed that function on the left side of equation does not make much sense. Symbol is just as good.

Reply:
Hm, I see, though we haven't explicitly talked about that